pypeit/core/wavecal/wv_fitting.py

```diff
@@ -214,15 +325,17 @@ def iterative_fitting(spec, tcent, ifit, IDs, llist, disp,
     #fmin, fmax = 0., 1.
     #xfit, yfit, wfit = tcent[ifit]/(nspec-1), all_ids[ifit], weights[ifit]
     xfit, yfit, wfit = tcent[ifit], all_ids[ifit], weights[ifit]
-    mask, fit = utils.robust_polyfit(xfit/xnspecmin1, yfit, n_order, function=func, sigma=sigrej_final,
-                                     minx=fmin, maxx=fmax, verbose=verbose, weights=wfit)#, debug=True)
-    irej = np.where(mask == 1)[0]
+    #mask, fit = utils.robust_polyfit(xfit/xnspecmin1, yfit, n_order, function=func, sigma=sigrej_final,
+    pypeitFit = fitting.robust_fit(xfit/xnspecmin1, yfit, n_order, function=func,
+                                   lower=sigrej_final, upper=sigrej_final,
+                                   minx=fmin, maxx=fmax, weights=wfit)#, debug=True)
+    irej = np.where(np.logical_not(pypeitFit.gpm))[0]
```

Comment:
This could be replaced by `np.where(pypeitFit.bpm)[0]` if you implement this as a `@property` of the DataContainer

Reply:
replaced `gpm` with `bool_gpm`